decidim-templates/app/views/decidim/templates/admin/proposal_answer_templates/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= t ".title" %>
+      <% if allowed_to?(:create, :template) %>
+        <%= link_to t("actions.new", scope: "decidim.admin", name: t("template.name", scope: "decidim.models").downcase), [:new, :proposal_answer_template], class: "button tiny button--title new" %>
+      <% end %>
+    </h2>
+  </div>
+  <div class="card-section">
+    <% if @templates.any? %>
+      <div class="table-scroll">
+        <table class="table-list questionnaire-templates">
+          <thead>
+          <tr>
+            <th><%= t("template.name", scope: "decidim.models") %></th>
+            <th><%= t("questionnaire_template.fields.title", scope: "decidim.models") %></th>
+            <th><%= t("questionnaire_template.fields.questions", scope: "decidim.models") %></th>
```

Comment:
Here we should have meaningful texts, these are for the questionnaires.
I guess it could be: "Proposal state" and "Available for"